src/mintpy/prep_hyp3.py

```diff
@@ -109,9 +109,13 @@ def add_hyp3_metadata(fname, meta, is_ifg=True):
     # HDF-EOS5 metadata
     if hyp3_meta['ReferenceGranule'].startswith('S1'):
         meta['beam_mode'] = 'IW'
+
+    meta['startUTC'] = date1.strftime('%Y-%m-%d %H:%M:%S.%f') + '+00:00'
+    meta['stopUTC'] = date2.strftime('%Y-%m-%d %H:%M:%S.%f') + '+00:00'
```

Comment:
[Here](https://github.com/insarlab/MintPy/blob/32645dd5df20ae30583ae0e4d528bcb71fd5fef6/src/mintpy/utils/isce_utils.py#L203-L205) is how we calculate the ASF frame number for Sentinel-1. So it seems like we need the ascending node time as well.

As for the current start/stopUTC, why add the `+00:00` at the end? Based on the example below, I would think the `*.strftime()` is enough.

```bash
(insar) yunjunz:~/data/test/FernandinaSenDT128/mintpy>$ info.py velocity.h5 
...
  startUTC                                        2014-12-13 11:49:44.367659
  stopUTC                                         2014-12-13 11:50:07.299273
...
```

Reply:
The `+00:00` just indicates that this timestamp is in the UTC timezone. If don't need that though, happy to remove it.

Reply:
I see. Yes, let's remove it for consistency please.